CHANGELOG.md

```diff
@@ -5,6 +5,12 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](http://keepachangelog.com/)
 and this project adheres to [Semantic Versioning](http://semver.org/).
 
+## [2.0.0-rc-9] - 2024-08-09
+
+### Changes
+
+- Bumped tests up to go1.22
```

Comment:
Goleak message should be fixed. If there is message, test should fail
https://github.com/avito-tech/go-transaction-manager/actions/runs/10226853987/job/28297721109?pr=107#step:8:85

Reply:
Resolved by `defer mongodbClient.Disconnect(ctx)`